wikipedia-sentences-incremental/app.py

```diff
@@ -79,19 +87,21 @@ def query(top_k):
             def ppr(x):
                 print_topk(x, text)
 
-            f.search_lines(
-                lines=[
-                    text,
-                ],
-                line_format='text',
-                on_done=ppr,
-                top_k=top_k,
-            )
+            with TimeContext(f'QPS: query', logger=f.logger):
```

Comment:
specify nr of docs you're querying with

Reply:
See https://github.com/jina-ai/examples/pull/460/commits/0d19b813ab8ac4446427508d138a56126f6a5b4b